docs/version0.17.md

```diff
@@ -92,10 +93,15 @@ The format of the `HOOKS` section of a Java dump, which shows internal VM event
 
 - Recorded times have been changed from milliseconds to microseconds to provide increased precision.
 - A new field, `3HKTOTALTIME`, is included, which gives the total duration of previous events.
-- The hook data is now reset after after each Java dump.
+- The hook data is now reset after each Java dump.
 
 For more information and an example of the new format, see [Java dump: HOOKS](dump_javadump.md#hooks)
 
+### LUDCL caching disabled by default
+
+By caching the Latest User Defined Class Loader (LUDCL), Java applications that use deserialization extensively can see a performance improvement. This
+capability is controlled by the [-Dcom.ibm.enableClassCaching](dcomibmenableclasscashing.md) system property and is now disabled by default. Earlier versions of the documentation incorrectly identified this property as disabled by default when it was actually enabled by default in the VM.
```

Comment:
Can you explicitly call out that prior to 0.17.0, the doc and VM were out of sync?  Even adding something like:
```
Earlier versions (prior to 0.17.0) of the documentation .....
```

Reply:
Updated accordingly